### PR TITLE
remove Create and Delete from no:loc

### LINKED ISSUE #20813: "Create" is blocked from localization
The term "Create" is blocked from localization with the no-loc metadata, but it may need to be localized in some cases such as "Create New" link, "Create" page/UI. If those UI terms should be left in English, then other UI such as "Save" and "Edit" may also need to be blocked from localization. Please help to check and update.

Similar issue/question with the term "Delete" in following article:
[https://github.com/dotnet/AspNetCore.Docs/blob/master/aspnetcore/tutorials/razor-pages/model.md](https://github.com/dotnet/AspNetCore.Docs/blob/master/aspnetcore/tutorials/razor-pages/model.md)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 39b6d0f3-9bc7-d091-0d45-9fc1e2e7aac8
* Version Independent ID: 20e7eb53-215b-09ab-205d-d20177221278
* Content: [Part 8, add validation](https://docs.microsoft.com/en-us/aspnet/core/tutorials/razor-pages/validation?view=aspnetcore-5.0&tabs=visual-studio)
* Content Source: [aspnetcore/tutorials/razor-pages/validation.md](https://github.com/dotnet/AspNetCore.Docs/blob/master/aspnetcore/tutorials/razor-pages/validation.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-tutorials**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #20813
cc @guardrex

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@scottaddie  `no-loc: [Create, Delete,` was only applied to these tutorials.

**Comment by guardrex** [comment]
I misunderstood the ask: I thought it was `no-loc`-ed at some higher level in the repo. I see now that it isn't and only requires removing the `no-loc` at topic level.

File: aspnetcore/tutorials/razor-pages/da1.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ author: rick-anderson
 description: Part 5 of tutorial series on Razor Pages.
 ms.author: riande
 ms.date: 09/20/2020
-no-loc: [Index, Create, Delete, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
+no-loc: [Index,  appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
```
Comment (guardrex) [inline aspnetcore/tutorials/razor-pages/da1.md:7]:
```suggestion
no-loc: [Index, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
```

File: aspnetcore/tutorials/razor-pages/model.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ author: rick-anderson
 description: Part 2 of tutorial series on Razor Pages. In this section, model classes are added.
 ms.author: riande
 ms.date: 11/11/2020
-no-loc: [Index, Create, Delete, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
+no-loc: [Index,  appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
```
Comment (guardrex) [inline aspnetcore/tutorials/razor-pages/model.md:7]:
```suggestion
no-loc: [Index, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
```

File: aspnetcore/tutorials/razor-pages/new-field.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ description: Part 7 of tutorial series on Razor Pages.
 ms.author: riande
 ms.custom: mvc
 ms.date: 09/28/2020
-no-loc: [Index, Create, Delete, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
+no-loc: [Index,  appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
```
Comment (guardrex) [inline aspnetcore/tutorials/razor-pages/new-field.md:8]:
```suggestion
no-loc: [Index, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
```

File: aspnetcore/tutorials/razor-pages/page.md [hunk 4]
```diff
@@ -4,7 +4,7 @@ author: rick-anderson
 description: Part 3 of tutorial series on Razor Pages.
 ms.author: riande
 ms.date: 09/25/2020
-no-loc: [Index, Create, Delete, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
+no-loc: [Index,  appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
```
Comment (guardrex) [inline aspnetcore/tutorials/razor-pages/page.md:7]:
```suggestion
no-loc: [Index, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
```

File: aspnetcore/tutorials/razor-pages/razor-pages-start.md [hunk 5]
```diff
@@ -4,7 +4,7 @@ author: rick-anderson
 description: This is the first tutorial of a series that teaches the basics of building an ASP.NET Core Razor Pages web app.
 ms.author: riande
 ms.date: 09/15/2020
-no-loc: [Index, Create, Delete, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
+no-loc: [Index,  appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
```
Comment (guardrex) [inline aspnetcore/tutorials/razor-pages/razor-pages-start.md:7]:
```suggestion
no-loc: [Index, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
```

File: aspnetcore/tutorials/razor-pages/search.md [hunk 6]
```diff
@@ -4,7 +4,7 @@ author: rick-anderson
 description: Part 6 of tutorial series on Razor Pages.
 ms.author: riande
 ms.date: 12/05/2019
-no-loc: [Index, Create, Delete, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
+no-loc: [Index,  appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
```
Comment (guardrex) [inline aspnetcore/tutorials/razor-pages/search.md:7]:
```suggestion
no-loc: [Index, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
```

File: aspnetcore/tutorials/razor-pages/sql.md [hunk 7]
```diff
@@ -4,7 +4,7 @@ author: rick-anderson
 description: Part 4 of tutorial series on Razor Pages.
 ms.author: riande
 ms.date: 09/26/2020
-no-loc: [Index, Create, Delete, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
+no-loc: [Index,  appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
```
Comment (guardrex) [inline aspnetcore/tutorials/razor-pages/sql.md:7]:
```suggestion
no-loc: [Index, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
```

File: aspnetcore/tutorials/razor-pages/validation.md [hunk 8]
```diff
@@ -5,7 +5,7 @@ description: Part 8 of tutorial series on Razor Pages.
 ms.author: riande
 ms.custom: mvc
 ms.date: 09/29/2020
-no-loc: [Index, Create, Delete, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
+no-loc: [Index,  appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
```
Comment (guardrex) [inline aspnetcore/tutorials/razor-pages/validation.md:8]:
```suggestion
no-loc: [Index, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
```

**Review by guardrex — APPROVED** [review]
Nit: Extra space

**Pull request merged**